core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/QcTarget.java

```diff
@@ -5,7 +5,11 @@
 import java.util.List;
 
 public enum QcTarget {
-  Library(QcCorrespondingField.NONE), Sample(QcCorrespondingField.NONE), Pool(QcCorrespondingField.NONE), Run(QcCorrespondingField.NONE), Container(QcCorrespondingField.NONE);
+  Library(QcCorrespondingField.NONE, QcCorrespondingField.CONCENTRATION, QcCorrespondingField.VOLUME), //
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)